backend/clubs/views.py

```diff
@@ -2599,6 +2615,10 @@ def create_tickets(self, request, *args, **kwargs):
 
         Ticket.objects.bulk_create(tickets)
 
+        order_limit = request.data.get("order_limit", 10)
```

Comment:
I'm looking at this logic and I'm a bit worried about our choice to delete tickets after creating ticket offerings:
We should figure out a way to freeze ticket counts, so that bought tickets can't be deleted. Maybe something like a "ticket drop time" as a time that the tickets will be frozen and not editable?

Not strictly related to your PR but just something that came to mind..

Reply:
Agreed, good catch. Resolving this for now -- will fix in another branch.